### DISCUSSION TITLE
Can't Open a Support Case

### DISCUSSION THREAD

**Discussion by nixxxo** [vercel/community#4592] [Help]
### Summary

Hello, 

I have been trying to submit the following case:

"Invoice number BF9DF637-0005
Date of issue October 23, 2023

Hello I was charged for a renewal of a domain that I did not aprove and have no intentions of using this domain, can I please get a refund?"

After I click submit it loads and nothing happens. I need help.

### Steps to Reproduce

Go to open a support case, select Billing and any Severe/Security Level. Click submit.

**Comment by amyegan** [comment]
Sorry you're having trouble with the support form, @nixxxo. I do see quite a few domain renewal billing tickets open for your account today and yesterday. They are currently open and awaiting billing support team availability. 🙇 

Tickets are generally processed by the order in which they are received, and someone from the billing support team will respond to yours as quickly as possible. Please do not open additional cases as this can create extra work for the team which slows response time.

**Reply by amyegan** [reply]
I checked with our engineering team about the form submission issue. They recommended checking your email, which should contain confirmations for each case you submitted, as a temporary workaround while we investigate the form confirmation problem
